a.core_concepts.md

```diff
@@ -196,25 +196,22 @@ kubectl get po nginx -o jsonpath='{.spec.containers[].image}{"\n"}'
 </p>
 </details>
 
-### Get nginx pod's ip created in previous step, use a temp busybox image to wget its '/'
+### Get nginx pod's IP created in previous step, use a temporary BusyBox image to retrieve the welcome page
 
 <details><summary>show</summary>
 <p>
 
 ```bash
 kubectl get po -o wide # get the IP, will be something like '10.1.1.131'
-# create a temp busybox pod
-kubectl run busybox --image=busybox --rm -it --restart=Never -- wget -O- 10.1.1.131:80
+# create a temporary BusyBox pod
+kubectl run busybox --image=busybox --rm -it --restart=Never -- wget -O- 10.1.1.131
 ```
 
 Alternatively you can also try a more advanced option:
 
 ```bash
-# Get IP of the nginx pod
-NGINX_IP=$(kubectl get pod nginx -o jsonpath='{.status.podIP}')
-# create a temp busybox pod
-kubectl run busybox --image=busybox --env="NGINX_IP=$NGINX_IP" --rm -it --restart=Never -- wget -O- $NGINX_IP:80
-``` 
+kubectl run busybox --image=busybox --restart=Never --rm -ti -- wget -O- $(kubectl get pod nginx -o jsonpath='{.status.podIP}{"\n"}')
```

Comment:
The `-env="NGINX_IP=$NGINX_IP` would have been redundant, ie this would work too:

```
kubectl run busybox --image=busybox  --rm -it --restart=Never -- wget -O- $NGINX_IP
```

But if we are going to have long one liners already, I thought this was not too advanced. I might be biased.

Reply:
Honestly, I'm not sure if it's easy to write jsonpath queries during the exam pressure. I'd leave it two lines, since it seems more clear for beginners.